src/libs/ValidationUtils.js

```diff
@@ -281,16 +282,11 @@ function validateIdentity(identity) {
 
 /**
  * @param {String} phoneNumber
- * @param {Boolean} [isCountryCodeOptional]
  * @returns {Boolean}
  */
-function isValidUSPhone(phoneNumber = '', isCountryCodeOptional) {
+function isValidUSPhone(phoneNumber = '') {
```

Comment:
This is wrong, first of all we are returning an object and not a boolean and we are not actually checking if the number is indeed a US phone. Also I'm not sure if we should be ignoring the `isCountryCodeOptional` the US phone numbers must be validated accordingly with and without country code.

Reply:
I didn't mean to revert. We should still keep the arguments as they are and use the lib to verify that the number is a US number.

Something like that:
```js
// add comment explaining that even if the country code is optional we need to explicity set if for use with lib
if isCountryCodeOptional
check if phone starts with +1 if not add the +1
endif
const parsedPhoneNumber = parsePhoneNumber(phoneNumber);
return parsedPhoneNumber.possible && parsedPhoneNumber.regionCode === CONST.COUNTRY.US;

```